testsupport/wait/host.go

```diff
@@ -1288,3 +1293,36 @@ func (a *HostAwaitility) GetHostOperatorPod() (corev1.Pod, error) {
 	}
 	return pods.Items[0], nil
 }
+
+// GetAPIProxyURL retrieves the api proxy route and returns its URL
+func (a *HostAwaitility) GetAPIProxyURL() string {
```

Comment:
I did try to use it initially but it requires an endpoint to return a status OK response but since the proxy doesn't have another endpoint it fails. I can change this when we have the health probe endpoint for the proxy defined. Actually, I'm putting that PR up today so maybe I can hold off on merging this one until after that. So I can switch to use `WaitForRouteToBeAvailable` once it's merged.

Reply:
The proxy health/readiness probe PR is up https://github.com/codeready-toolchain/registration-service/pull/230

Reply:
never mind - it's part of the reg-service deployment and before it gets the proxy URL it also waits until the reg-service is there anyway.

Reply:
but the health/readiness probe will help for sure :+1: